src/plugins/timeConductor/ConductorHistory.vue

```diff
@@ -135,10 +138,20 @@ export default {
     methods: {
         getHistoryMenuItems() {
             const history = this.historyForCurrentTimeSystem.map(timespan => {
```

Comment:
I do see some duplicate values, may be we can clean up that in this or create new issue.

can convert `historyForCurrentTimeSystem` to Array `Set `

Reply:
good call, might as well do it in this one since I'm in that exact code...